ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1393 @@
+{
```

Comment:
<p>You should move those even further - to a separate python module</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>reworked</p>